Automattic/sensei
# Refine answer feedback blocks

Follow up to #4313 

### Changes proposed in this Pull Request

* Rework the question block's inner blocks a bit to also include answers, so the feedback blocks can go below them
* Remove inserter & always have feedback blocks, but make them collapsible
* Update feedback block style to look the same as the frontend

### Testing instructions

* Open a question or a quiz with questions that have answer feedback
* Make sure that feedback is converted to blocks
* Update correct/incorrect feedback, make sure everything works properly


### Screenshot / Video

<img width="682" alt="image" src="https://user-images.githubusercontent.com/176949/140228447-9da6af01-3787-4578-90cd-43eb34aebfec.png">
